Improve documentation about how to import in Typescript+Node<13.11

Hi,  im trying to use Hashids with Typescript and Node.

`import Hashids from 'hashids';` 

I'm not getting any Typescript errors in the editor, but Im getting this error when the code is compiled. 

`Using ts-node version 8.5.4, typescript version 3.7.5`

`Error: No valid exports main found for 'node_modules\hashids'
    at resolveExportsTarget (internal/modules/cjs/loader.js:625:9)
    at applyExports (internal/modules/cjs/loader.js:502:14)
    at resolveExports (internal/modules/cjs/loader.js:551:12)
    at Function.Module._findPath (internal/modules/cjs/loader.js:657:22)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:960:27)
    at Function.Module._load (internal/modules/cjs/loader.js:855:27)
    at Module.require (internal/modules/cjs/loader.js:1033:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (encodeID.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:1144:30)
[ERROR] 18:03:43 Error: No valid exports main found for 'node_modules\hashids'`

Thanks!

--- a/README.md
+++ b/README.md
@@ -55,7 +55,20 @@ Note: When using Node that supports [conditional exports](https://nodejs.org/api
 
 #### Use in **TypeScript**:
 
-Import or require based on the environment (see above).
+Import or require based on the environment (see above). If you want to use the CommonJS module syntax (`require`), you'll need to install the Node.js types from the `DefinitelyTyped` repository.
+
+```
+npm install @types/node
+```
+
+If you want to use the ESM syntax (`import`), you will need to include the following options in your `tsconfig.json`.
+
+```json
+{
+  "allowSyntheticDefaultImports": true,
+  "esModuleInterop": true
+}
+```
 
 If you get errors stating: `Cannot find name 'BigInt'`, add [`"esnext.bigint"`](https://github.com/microsoft/TypeScript/blob/master/src/lib/esnext.bigint.d.ts) or [`"esnext"`](https://github.com/microsoft/TypeScript/blob/master/src/lib/esnext.d.ts) to your `tsconfig.json` file, under `"lib"`:
 

